### ISSUE TITLE
SickChill wont start after version 20210729-2 was installt on synolgy Dsm 6.2.4-25556

### ISSUE THREAD

**Issue by xmenxmen** [SynoCommunity/spksrc#4856]
### Setup
_Package Name:_ SickChill 
_Package Version:_ version 20210729-2

_NAS Model:_ DS5113+
_NAS Architecture:_ 
_DSM version:_  6.2.4-25556

### Expected behavior
Tell us what should happen

### Actual behavior start
Tell us what happens instead It start and stop 1 min after

### Steps to reproduce
_1._ 
_2._ 
_3._ 

### Setup
_Package Name:_ 
_Package Version:_ 

_NAS Model:_ 
_NAS Architecture:_ 
_DSM version:_ 

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### Steps to reproduce
_1._ 
_2._ 
_3._ 

#### Package log
_Installer log: find logfile `/var/log/packages/{package}.log`, for DSM<6: `/var/packages/{package}/target/var/{package}_installer.log` and protocol in Package Center.
You find service log and other log files in `/var/packages/{package}/target/var`.
_
```
Insert the package log(s) here
```
#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
```

**Comment by ejc1967** [comment]
Hi,

I encountered the same problem, tried upgrading from 20210329-1, as well as a clean install.
Both times SickChill starts, but stop again after less than a minute.
Only ERROR I could find is this: ERROR: SQLAlchemy-1.3.20-cp37-none-any.whl is not a supported wheel on this platform.
See included partial package log from the time of the update and clean install.
After reinstalling 20210329-1 and manually fixing the small env issue, SickChill works just fine again.

**Setup**
Package Name: SickChill
Package Version: version 20210729-2

NAS Model: DS3612xs
NAS Architecture: Bromolow (x64)
DSM version: DSM 6.2.4-25556 Update 2

**Package log**

2021/09/10 07:14:02	upgrade sickchill 20210329-1 Begin start-stop-status stop
Stopping sickchill ...
2021/09/10 07:14:22	upgrade sickchill 20210329-1 End start-stop-status stop ret=[0]
2021/09/10 07:14:33	upgrade sickchill 20210329-1 Begin preupgrade
2021/09/10 07:14:33	===> Step preupgrade. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:14:33	Begin service_preupgrade
2021/09/10 07:14:33	Removing old /volume1/@appstore/sickchill/env for new Python 3.8
2021/09/10 07:14:35	End service_preupgrade
2021/09/10 07:15:27	upgrade sickchill 20210329-1 End preupgrade ret=[0]
2021/09/10 07:15:27	upgrade sickchill 20210329-1 Begin preuninst
2021/09/10 07:15:27	upgrade sickchill 20210329-1 End preuninst ret=[0]
2021/09/10 07:15:27	upgrade sickchill 20210329-1 Begin /bin/rm -rf /volume1/@appstore/sickchill
2021/09/10 07:15:31	upgrade sickchill 20210329-1 End /bin/rm -rf /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:15:31	upgrade sickchill 20210329-1 Begin postuninst
2021/09/10 07:15:31	upgrade sickchill 20210329-1 End postuninst ret=[0]
2021/09/10 07:15:33	upgrade sickchill 20210729-2 Begin preinst
2021/09/10 07:15:33	===> Step preinst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:15:33	upgrade sickchill 20210729-2 End preinst ret=[0]
2021/09/10 07:15:33	upgrade sickchill 20210729-2 Begin /bin/rm -rf /volume1/@appstore/sickchill
2021/09/10 07:15:33	upgrade sickchill 20210729-2 End /bin/rm -rf /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:15:33	upgrade sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/sickchill
2021/09/10 07:15:33	upgrade sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/mkdir -p /var/packages/sickchill
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/mkdir -p /var/packages/sickchill ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/sickchill/INFO
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/sickchill/INFO ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/scripts
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/scripts ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/sickchill/scripts
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/sickchill/scripts ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/WIZARD_UIFILES
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/WIZARD_UIFILES ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/sickchill/WIZARD_UIFILES
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/sickchill/WIZARD_UIFILES ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/conf
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/conf ret=[0]
2021/09/10 07:15:34	upgrade sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/sickchill/conf
2021/09/10 07:15:34	upgrade sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/sickchill/conf ret=[0]
2021/09/10 07:15:35	upgrade sickchill 20210729-2 Begin postinst
2021/09/10 07:15:35	===> Step postinst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:15:35	Begin service_postinst
2021/09/10 07:15:39	created virtual environment CPython3.8.11.final.0-64 in 1565ms
2021/09/10 07:15:39	  creator CPython3Posix(dest=/volume1/@appstore/sickchill/env, clear=False, no_vcs_ignore=False, global=True)
2021/09/10 07:15:39	  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/root/.local/share/virtualenv)
2021/09/10 07:15:39	    added seed packages: pip==21.1.1, setuptools==56.0.0, wheel==0.36.2
2021/09/10 07:15:39	  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
2021/09/10 07:15:42	Looking in links: /volume1/@appstore/sickchill/share/wheelhouse
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/CacheControl-0.12.6-py2.py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/Deprecated-1.2.10-py2.py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/IMDbPY-2020.9.25-py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/Js2Py-0.70-py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/Mako-1.1.3-py2.py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/MarkupSafe-1.1.1-py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyGithub-1.53-py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyJWT-1.7.1-py2.py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyNMA-1.0-py3-none-any.whl
2021/09/10 07:15:42	Processing /volume1/@appstore/sickchill/share/wheelhouse/PySocks-1.7.1-py3-none-any.whl
2021/09/10 07:15:42	ERROR: SQLAlchemy-1.3.20-cp37-none-any.whl is not a supported wheel on this platform.
2021/09/10 07:15:42	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill
2021/09/10 07:15:51	End service_postinst
2021/09/10 07:15:51	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill/var
2021/09/10 07:15:51	upgrade sickchill 20210729-2 End postinst ret=[0]
2021/09/10 07:15:51	upgrade sickchill 20210729-2 Begin postupgrade
2021/09/10 07:15:51	===> Step postupgrade. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:16:20	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill/var
2021/09/10 07:16:20	Begin service_postupgrade
2021/09/10 07:16:20	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill
2021/09/10 07:16:20	End service_postupgrade
2021/09/10 07:16:20	upgrade sickchill 20210729-2 End postupgrade ret=[0]
2021/09/10 07:16:21	upgrade sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:16:22	upgrade sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:16:23	upgrade sickchill 20210729-2 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2021/09/10 07:16:23	upgrade sickchill 20210729-2 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2021/09/10 07:17:26	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:17:26	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:17:27	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:17:28	start sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:18:47	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:18:47	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:18:48	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:18:49	start sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:22:14	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:22:14	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:22:14	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:22:15	start sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:23:28	uninstall sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:23:28	uninstall sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:23:28	uninstall sickchill 20210729-2 Begin preuninst
2021/09/10 07:23:28	===> Step preuninst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:23:28	uninstall sickchill 20210729-2 End preuninst ret=[0]
2021/09/10 07:23:28	uninstall sickchill 20210729-2 Begin /bin/rm -rf /volume1/@appstore/sickchill
2021/09/10 07:23:29	uninstall sickchill 20210729-2 End /bin/rm -rf /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:23:29	uninstall sickchill 20210729-2 Begin postuninst
2021/09/10 07:23:29	===> Step postuninst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:23:29	Removing user sc-sickchill
2021/09/10 07:23:30	uninstall sickchill 20210729-2 End postuninst ret=[0]
2021/09/10 07:24:35	install sickchill 20210729-2 Begin preinst
2021/09/10 07:24:35	===> Step preinst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:24:35	install sickchill 20210729-2 End preinst ret=[0]
2021/09/10 07:24:35	install sickchill 20210729-2 Begin /bin/rm -rf /volume1/@appstore/sickchill
2021/09/10 07:24:35	install sickchill 20210729-2 End /bin/rm -rf /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:24:35	install sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/sickchill
2021/09/10 07:24:35	install sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/sickchill ret=[0]
2021/09/10 07:24:35	install sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill
2021/09/10 07:24:35	install sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/mkdir -p /var/packages/sickchill
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/mkdir -p /var/packages/sickchill ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/sickchill/INFO
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/sickchill/INFO ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/scripts
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/scripts ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/sickchill/scripts
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/sickchill/scripts ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/WIZARD_UIFILES
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/WIZARD_UIFILES ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/sickchill/WIZARD_UIFILES
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/sickchill/WIZARD_UIFILES ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/rm -rf /var/packages/sickchill/conf
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/rm -rf /var/packages/sickchill/conf ret=[0]
2021/09/10 07:24:36	install sickchill 20210729-2 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/sickchill/conf
2021/09/10 07:24:36	install sickchill 20210729-2 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/sickchill/conf ret=[0]
2021/09/10 07:24:37	install sickchill 20210729-2 Begin postinst
2021/09/10 07:24:37	===> Step postinst. USER=sickchill GROUP=sc-download SHARE_PATH=
2021/09/10 07:24:37	Adding 'sc-sickchill' to 'sc-download'
2021/09/10 07:24:38	Group Name: [sc-download]
2021/09/10 07:24:38	Group Type: [AUTH_LOCAL]
2021/09/10 07:24:38	Group ID:   [65549]
2021/09/10 07:24:38	Group Members: 
2021/09/10 07:24:38	0:[sc-medusa]
2021/09/10 07:24:38	1:[sc-sabnzbd]
2021/09/10 07:24:38	2:[sc-nzbdrone]
2021/09/10 07:24:38	3:[sc-sickchill]
2021/09/10 07:24:38	Begin service_postinst
2021/09/10 07:24:38	created virtual environment CPython3.8.11.final.0-64 in 234ms
2021/09/10 07:24:38	  creator CPython3Posix(dest=/volume1/@appstore/sickchill/env, clear=False, no_vcs_ignore=False, global=True)
2021/09/10 07:24:38	  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/root/.local/share/virtualenv)
2021/09/10 07:24:38	    added seed packages: pip==21.1.1, setuptools==56.0.0, wheel==0.36.2
2021/09/10 07:24:38	  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
2021/09/10 07:24:40	Looking in links: /volume1/@appstore/sickchill/share/wheelhouse
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/CacheControl-0.12.6-py2.py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/Deprecated-1.2.10-py2.py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/IMDbPY-2020.9.25-py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/Js2Py-0.70-py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/Mako-1.1.3-py2.py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/MarkupSafe-1.1.1-py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyGithub-1.53-py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyJWT-1.7.1-py2.py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/PyNMA-1.0-py3-none-any.whl
2021/09/10 07:24:40	Processing /volume1/@appstore/sickchill/share/wheelhouse/PySocks-1.7.1-py3-none-any.whl
2021/09/10 07:24:40	ERROR: SQLAlchemy-1.3.20-cp37-none-any.whl is not a supported wheel on this platform.
2021/09/10 07:24:40	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill
2021/09/10 07:24:45	End service_postinst
2021/09/10 07:24:45	Granting 'sc-sickchill' unix ownership on /volume1/@appstore/sickchill/var
2021/09/10 07:24:45	install sickchill 20210729-2 End postinst ret=[0]
2021/09/10 07:24:45	install sickchill 20210729-2 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2021/09/10 07:24:45	install sickchill 20210729-2 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2021/09/10 07:24:46	install sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:24:48	install sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:27:33	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:27:33	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:27:33	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:27:34	start sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:28:20	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:28:20	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:28:20	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:28:21	start sickchill 20210729-2 End start-stop-status start ret=[0]
2021/09/10 07:29:00	start sickchill 20210729-2 Begin start-stop-status stop
2021/09/10 07:29:00	start sickchill 20210729-2 End start-stop-status stop ret=[0]
2021/09/10 07:29:01	start sickchill 20210729-2 Begin start-stop-status start
2021/09/10 07:29:02	start sickchill 20210729-2 End start-stop-status start ret=[0]

**Comment by publicarray** [comment]
What does the `var/packages/sickchill/target/var/sickchill.log` log file show?
```
$ cat /var/packages/sickchill/target/var/sickchill.log
Sun Sep 12 23:20:37 AEST 2021
Starting sickchill command /volume1/@appstore/sickchill/env/bin/python3 /volume1/@appstore/sickchill/share/SickChill/SickChill.py --daemon --nolaunch --pidfile /volume1/@appstore/sickchill/var/sickchill.pid --config /volume1/@appstore/sickchill/var/data/config.ini --datadir /volume1/@appstore/sickchill/var/data 
Writing PID: 24219 to /volume1/@appstore/sickchill/var/sickchill.pid
```

**Comment by publicarray** [comment]
The version I compiled a while ago works, but the online version doesn't. Would you be willing to confirm this? Here is what I used to test before publishing. https://seby.io/download/sickchill_noarch-dsm6_20210729-2.spk

**Comment by ejc1967** [comment]
> The version I compiled a while ago works, but the online version doesn't. Would you be willing to confirm this? Here is what I used to test before publishing. https://seby.io/download/sickchill_noarch-dsm6_20210729-2.spk

I can confirm that this package is working for me as well !

**Comment by BKSteve** [comment]
There is a little issue with the core SickChill code with a work around [https://github.com/SickChill/SickChill/discussions/7364#discussion-3477151](https://github.com/SickChill/SickChill/discussions/7364#discussion-3477151)

**Comment by Wismerhull** [comment]
> The version I compiled a while ago works, but the online version doesn't. Would you be willing to confirm this? Here is what I used to test before publishing. https://seby.io/download/sickchill_noarch-dsm6_20210729-2.spk

Worked for me, just takes 5 mins to load the GUI

**Comment by BKSteve** [comment]
> 
> 
> The version I compiled a while ago works, but the online version doesn't. Would you be willing to confirm this? Here is what I used to test before publishing. https://seby.io/download/sickchill_noarch-dsm6_20210729-2.spk

Now confirmed that the linked version runs out of the box on both DS412+ and DS1812+ with no need for init_helpers.py per SickChill #7364 
It also loads and runs in 5 minutes and not the 10+ the online package takes to load on 412+

**Comment by djpoulet** [comment]
> The version I compiled a while ago works, but the online version doesn't. Would you be willing to confirm this? Here is what I used to test before publishing. https://seby.io/download/sickchill_noarch-dsm6_20210729-2.spk

I just tried this version on my DS216j and it doesn't work either.
Can you provide a link for the previous version of the package, it was working? Thank's

**Comment by BKSteve** [comment]
@djpoulet  you need to be little more specific On didn't work. Got any logs? Did the**still not working** in SickChill #7364?

**Comment by BKSteve** [comment]
The linked package above is certainly cleaner and faster at restarting than the one that was online.

There does seem to be issues with ARM v5 and v7 ARMada38x machines that have reported issues in 7504.

Here's a snippet:
```
Command result:   ERROR: Command errored out with exit status 1:\
   command: /volume1/@appstore/sickchill/env/bin/python3 /tmp/pip-standalone-pip-13lwvnxn/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-c_t1w1fk/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple --find-links /volume1/@appstore/sickchill/share/wheelhouse -- 'setuptools>=40.6.0' wheel 'cffi>=1.12; platform_python_implementation != '"'"'PyPy'"'"'' 'setuptools-rust>=0.11.4'\
        cwd: None\
...
  Building wheels for collected packages: cffi\
    Building wheel for cffi (setup.py): started\
    Building wheel for cffi (setup.py): finished with status 'error'\
    ERROR: Command errored out with exit status 1:\
     command: /volume1/@appstore/sickchill/env/bin/python3 -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-g092ncbx/cffi_4b61de02b6c047689b1894a1f396744a/setup.py'"'"'; __file__='"'"'/tmp/pip-install-g092ncbx/cffi_4b61de02b6c047689b1894a1f396744a/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\\r\\n'"'"', '"'"'\\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-max2i7ns\
         cwd: /tmp/pip-install-g092ncbx/cffi_4b61de02b6c047689b1894a1f396744a/\
    Complete output (48 lines):\
    unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory\
    unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory\
```

**Comment by djpoulet** [comment]
@BKSteve oups sorry if I was not precise enough.
Seems like you reply on this other thread. Thanks
https://github.com/SickChill/SickChill/issues/7504#issuecomment-918850566

**Comment by BKSteve** [comment]
There are a number of issues with ARM7 based devices and installation as I need some help to identify the ARM7 install issue.
[https://github.com/SickChill/SickChill/issues/7504#issuecomment-921141827](https://github.com/SickChill/SickChill/issues/7504#issuecomment-921141827)
The log shows issues with cffi in the log file of the linked it's line 83
`unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory`

here's a few lines preceding and up to the fail message.
```
Found wheelhouse dir at /volume1/@appstore/sickchill/share/wheelhouse
pip args: /volume1/@appstore/sickchill/env/bin/python3 -m pip install --no-input --disable-pip-version-check --no-python-version-warning --no-color -qU -f/volume1/@appstore/sickchill/share/wheelhouse appdirs==1.4.4 babelfish==0.6.0b1; python_version >= "3.6" and python_version < "4.0" beautifulsoup4==4.9.3 beekeeper-alt==2021.7.16; python_full_version >= "3.6.2" and python_version < "4" bencode.py==4.0.0 cachecontrol==0.12.6; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") certifi==2021.5.30; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" cffi==1.14.6; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" or python_full_version >= "3.5.0" and python_version >= "3.6" chardet==4.0.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") charset-normalizer==2.0.3; python_full_version >= "3.6.0" and python_version >= "3" click==8.0.1; python_version >= "3.6" cloudscraper==1.2.58 colorama==0.4.4; python_version >= "3.6" and python_full_version < "3.0.0" and platform_system == "Windows" or platform_system == "Windows" and python_version >= "3.6" and python_full_version >= "3.5.0" configobj==5.0.6 cryptography==3.4.7; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6" decorator==5.0.9; python_version >= "3.5" deluge-client==1.9.0 deprecated==1.2.12; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" dogpile.cache==1.1.3; python_version >= "3.6" enzyme==0.4.1 feedparser==6.0.8; python_version >= "3.6" future==0.18.2; python_version >= "2.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" gntp==1.0.3 greenlet==1.1.0; python_version >= "3" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3" guessit==3.3.1 html5lib==1.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") httplib2==0.19.1 idna==3.2; python_version >= "3.5" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.5" ifaddr==0.1.7 imagesize==1.2.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") imdbpy==2021.4.18 importlib-metadata==1.7.0; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "3.8" or python_version < "3.8" and python_version >= "3.6" and python_full_version >= "3.5.0" or python_version >= "2.7" and python_full_version < "3.0.0" and python_version < "3.8" or python_full_version >= "3.6.0" and python_version < "3.8" ipaddress==1.0.23 js2py==0.71 jsonrpclib-pelix==0.4.2 kodipydent-alt==2021.7.16; python_full_version >= "3.6.2" and python_version < "4" lxml==4.6.3; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" mako==1.1.4; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") markdown2==2.4.0; python_version >= "3.5" and python_version < "4" markupsafe==2.0.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" msgpack==1.0.2; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" new-rtorrent-python==1.0.1a0 oauthlib==3.1.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" packaging==20.9; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") pbr==5.6.0; python_version >= "3.6" profilehooks==1.12.0 putio.py==8.7.0 pyaes==1.6.1 pycparser==2.20; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" pygithub==1.55; python_version >= "3.6" pyjsparser==2.7.1 pyjwt==2.1.0; python_version >= "3.6" pymediainfo==5.1.0; python_version >= "3.6" pynacl==1.4.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" pynma==1.0 pyopenssl==20.0.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") pyparsing==2.4.7; python_version >= "2.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" or python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" pysocks==1.7.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") pysrt==1.1.2 python-dateutil==2.8.2; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.3.0") python-slugify==5.0.2; python_version >= "3.6" python-twitter==3.5 python3-fanart==2.0.0 pytz==2021.1 qbittorrent-api==2021.5.22 rarfile==4.0 rebulk==3.0.1 requests-oauthlib==1.3.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" requests-toolbelt==0.9.1 requests==2.26.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0") send2trash==1.7.1 sgmllib3k==1.0.0; python_version >= "3.6" six==1.16.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" or python_full_version >= "3.5.0" or python_version >= "3.4" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.4" or python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" soupsieve==2.2.1; python_version >= "3.6" sqlalchemy==1.4.22; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0") stevedore==3.3.0; python_version >= "3.6" subliminal==2.1.0 text-unidecode==1.3; python_version >= "3.6" timeago==1.0.15 tmdbsimple==2.8.0 tornado==6.1; python_version >= "3.5" tus.py==1.3.4 tvdbsimple==1.0.6 tzlocal==2.1 unidecode==1.2.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") urllib3==1.26.6; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4" or python_full_version >= "3.6.0" and python_version < "4" validators==0.18.2; python_version >= "3.4" webencodings==0.5.1; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" win-inet-pton==1.1.0 wrapt==1.12.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" xmltodict==0.12.0; python_full_version >= "3.6.2" and python_version < "4" zipp==3.5.0; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "3.8" or python_version < "3.8" and python_version >= "3.6" and python_full_version >= "3.5.0"
Command result:   ERROR: Command errored out with exit status 1:
   command: /volume1/@appstore/sickchill/env/bin/python3 /tmp/pip-standalone-pip-94f_n1o7/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-20r7mqgg/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple --find-links /volume1/@appstore/sickchill/share/wheelhouse -- 'setuptools>=40.6.0' wheel 'cffi>=1.12; platform_python_implementation != '"'"'PyPy'"'"'' 'setuptools-rust>=0.11.4'
       cwd: None
  Complete output (131 lines):
  Looking in links: /volume1/@appstore/sickchill/share/wheelhouse
  Collecting setuptools>=40.6.0
    Using cached setuptools-58.0.4-py3-none-any.whl (816 kB)
  Collecting wheel
    Using cached wheel-0.37.0-py2.py3-none-any.whl (35 kB)
  Collecting cffi>=1.12
    Using cached cffi-1.14.6.tar.gz (475 kB)
  Collecting setuptools-rust>=0.11.4
    Using cached setuptools_rust-0.12.1-py3-none-any.whl (22 kB)
  Collecting pycparser
    Using cached pycparser-2.20-py2.py3-none-any.whl (112 kB)
  Collecting toml>=0.9.0
    Using cached toml-0.10.2-py2.py3-none-any.whl (16 kB)
  Collecting semantic-version>=2.6.0
    Using cached semantic_version-2.8.5-py2.py3-none-any.whl (15 kB)
  Building wheels for collected packages: cffi
    Building wheel for cffi (setup.py): started
    Building wheel for cffi (setup.py): finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /volume1/@appstore/sickchill/env/bin/python3 -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-zgtq2efj/cffi_eaeb3e41d581497c9469c62aeb57fee2/setup.py'"'"'; __file__='"'"'/tmp/pip-install-zgtq2efj/cffi_eaeb3e41d581497c9469c62aeb57fee2/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-qtpaosdq
         cwd: /tmp/pip-install-zgtq2efj/cffi_eaeb3e41d581497c9469c62aeb57fee2/
    Complete output (48 lines):
    unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory
    unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory

        No working compiler found, or bogus compiler options passed to
        the compiler from Python's standard "distutils" module.  See
        the error messages above.  Likely, the problem is not related
        to CFFI but generic to the setup.py of any Python package that
        tries to compile C code.  (Hints: on OS/X 10.8, for errors about
        -mno-fused-madd see http://stackoverflow.com/questions/22313407/
        Otherwise, see https://wiki.python.org/moin/CompLangPython or
        the IRC channel #python on irc.libera.chat.)

        Trying to continue anyway.  If you are trying to install CFFI from
        a build done in a different context, you can ignore this warning.

    running bdist_wheel
    running build
    running build_py
    creating build
    creating build/lib.linux-armv7l-3.8
    creating build/lib.linux-armv7l-3.8/cffi
    copying cffi/verifier.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/vengine_gen.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/vengine_cpy.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/setuptools_ext.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/recompiler.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/pkgconfig.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/model.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/lock.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/ffiplatform.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/error.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/cparser.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/commontypes.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/cffi_opcode.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/backend_ctypes.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/api.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/__init__.py -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/_cffi_include.h -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/parse_c_type.h -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/_embedding.h -> build/lib.linux-armv7l-3.8/cffi
    copying cffi/_cffi_errors.h -> build/lib.linux-armv7l-3.8/cffi
    running build_ext
    building '_cffi_backend' extension
    creating build/temp.linux-armv7l-3.8
    creating build/temp.linux-armv7l-3.8/c
    /spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/arm-unknown-linux-gnueabi/sysroot/usr/include -D__ARM_PCS_VFP=1 -I/spksrc/spk/python38/work-armv7-6.1/install//var/packages/python38/target/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -L /spksrc/spk/python38/work-armv7-6.1/install/var/packages/python38/target/lib -I /spksrc/spk/python38/work-armv7-6.1/install/var/packages/python38/target/include -I/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/arm-unknown-linux-gnueabi/sysroot/usr/include -D__ARM_PCS_VFP=1 -I/spksrc/spk/python38/work-armv7-6.1/install//var/packages/python38/target/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -L /spksrc/spk/python38/work-armv7-6.1/install/var/packages/python38/target/lib -I /spksrc/spk/python38/work-armv7-6.1/install/var/packages/python38/target/include -fPIC -I/usr/include/ffi -I/usr/include/libffi -I/volume1/@appstore/sickchill/env/include -I/var/packages/python38/target/include/python3.8 -c c/_cffi_backend.c -o build/temp.linux-armv7l-3.8/c/_cffi_backend.o
    unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory
    error: command '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc' failed with exit status 1
    ----------------------------------------
    ERROR: Failed building wheel for cffi
    Running setup.py clean for cffi
  Failed to build cffi
```
grateful for any suggestions and what is needed to have a successful ARM7 based DSM install.

**Comment by BKSteve** [comment]
The errors continue and I I need some help with why cffi crashes on wheel build on arm devices.

[https://github.com/SickChill/SickChill/issues/7504#issuecomment-922227426](https://github.com/SickChill/SickChill/issues/7504#issuecomment-922227426)

```
unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory
unable to execute '/spksrc/toolchain/syno-armv7-6.1/work/arm-unknown-linux-gnueabi/bin/arm-unknown-linux-gnueabi-gcc': No such file or directory
```

**Comment by publicarray** [comment]
@BKSteve if I understand correctly the pip package needs a compiler to work. This needs to be done beforehand (cross compiled) and sipped as a wheel.

```
  Building wheels for collected packages: cffi
    Building wheel for cffi (setup.py): started
    Building wheel for cffi (setup.py): finished with status 'error'
```

Aka it's not using the cached version

**Comment by BKSteve** [comment]
Yes, just built now with cross/cffi and build_depends cross/python38
My issue is also I don't have a DSM with arm architecture to test on. Mine are all x64 based which install.

**Comment by publicarray** [comment]
Yea, I don't either (all are x64 too).

**Comment by BKSteve** [comment]
There are 2 people with arm devices that have been helping by posting logs, so I just have to wait.

**Comment by djpoulet** [comment]
@BKSteve I’ve seen your messages for testing. On week-end with family, try to test in a few hours or tomorrow (GMT +2).

**Comment by BKSteve** [comment]
Need some help with my builds of this package. With edits it now compiles 8/11 packages successfully but the qoriq and evansport (both 6&7) throw `Crossenv has leaked into another Python interpreter!` when the make arch is installing cffi.
```
Building wheels for collected packages: cffi
  Building wheel for cffi (setup.py): started
  Building wheel for cffi (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /spksrc/spk/sickchill/work-evansport-6.1/crossenv/cross/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-9x9jz12m/cffi_ece2816d91014f85b99776d1d145320b/setup.py'"'"'; __file__='"'"'/tmp/pip-install-9x9jz12m/cffi_ece2816d91014f85b99776d1d145320b/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-devap2j3
       cwd: /tmp/pip-install-9x9jz12m/cffi_ece2816d91014f85b99776d1d145320b/
  Complete output (13 lines):
  *******************************************************
  * Crossenv has leaked into another Python interpreter!
  * You should probably file a bug report.
  * Version 3.8.11 (default, Sep 27 2021, 09:47:46)
  [GCC 8.3.0]
  * Executable /spksrc/spk/sickchill/work-evansport-6.1/crossenv/cross/bin/python
  *******************************************************
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for cffi
  Running setup.py clean for cffi
Failed to build cffi
```
[build-evansport-6.1.log.zip](https://github.com/SynoCommunity/spksrc/files/7266816/build-evansport-6.1.log.zip)

**Comment by publicarray** [comment]
Should be fixed in #4920 (Thanks @BKSteve), If not please open a new issue since the build has some major changes.

https://synocommunity.com/package/sickchill

**Comment by mediafreakch** [comment]
@BKSteve I can help testing too, since neither of the packages at https://synocommunity.com/ work for me. I am on `Linux 2.6.32.12, armv5tel GNU/Linux synology_88f6282_213air`.

**Comment by BKSteve** [comment]
@mediafreakch  come to [7504](https://github.com/SickChill/SickChill/issues/7504#issuecomment-964225852) On SickChill where there's links to test builds and some issues with the current released v-3

**Comment by mediafreakch** [comment]
Right, `20210729-3` does not work on my Synology. Same symptoms as described elsewhere: After starting the packe GUI is not accessible and after some time (5-10') the package shows as stopped.
